pkg/codegen/utilities.go

```diff
@@ -106,3 +107,25 @@ func CleanDir(dirPath string, exclusions StringSet) error {
 
 	return nil
 }
+
+var commonEnumNameReplacements = map[string]string{
+	"*": "Asterisk",
+	"0": "Zero",
+	"1": "One",
+	"2": "Two",
+	"3": "Three",
+	"4": "Four",
+	"5": "Five",
+	"6": "Six",
+	"7": "Seven",
+	"8": "Eight",
+	"9": "Nine",
+}
+
+func ReplaceInvalidEnumName(name string) (string, error) {
```

Comment:
It could be me, but this function feels slightly odd on its own. Would it be possible to extract the full `makeSafeEnumName` function, which seems more self-sufficient?

Reply:
That's not quite as simple. Each language has its own idea of what is a valid identifier. And each language has its own representation of what an enum name should look like (like all caps for python, `Equals` having special meaning in C#, etc.). I tried to extract what was reasonably extractable.

Reply:
Yes, I was too optimistic. The following would be slightly more apealing to me but it's up to your judgment:
1. Call it something like `ExpandShorEnumName`.
2. Never return an error, just the original string or the expanded string.
3. Call it from `makeSafeEnumName` before the `if`. 
4. `if len(safeName) == 1 && !isLegalIdentifierStart(rune(safeName[0]))` always returns an error if `true`.

Reply:
I like that, will change it up.